api/reporting/docs/cohort_definition_schema.json

```diff
@@ -0,0 +1,201 @@
+          "description": "A collection of operands",
+          "type": "array",
+          "items": [
+            {
+              "description": "An operand",
+              "type": "integer"
+            }
+          ]
+        }
+      },
+      "required": ["name", "operator", "operands"]
+    },
+    "search_group": {
+      "description": "A search group is a collection of concepts which can be 'OR'ed together",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "A unique identifier for the search group",
+          "type": "string"
+        },
+        "items": {
+          "description": "A collection of search group items",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/search_group_item"
+            }
+          ]
+        },
+        "temporal": {
+          "description": "Is the search group temporal(relating groups within a time frame). ",
+          "type": "boolean"
+        }
+      },
+      "required": ["id", "items", "temporal"]
+    },
+    "search_group_item": {
+      "description": "A search group item is a collection of concepts of the same domain which can be 'OR'ed together",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The unique identifier for the search group item",
+          "type": "string"
+        },
+        "type": {
+          "description": "The description for this search group item",
+          "type": "string"
+        },
+        "searchParameters": {
+          "description": "A collection search parameters",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/search_parameter"
+            }
+          ]
+        },
+        "modifiers": {
+          "description": "A collection of modifiers that refine the search",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/modifier"
+            }
+          ]
+        },
+        "temporalGroup": {
+          "description": "Temporal group index for the search group item",
+          "type": "integer"
+        }
+      },
+      "required": ["id", "type", "searchParameters", "modifiers"]
+    },
+    "search_parameter": {
+      "description": "A search parameter contains metadata about a specific concept",
+      "type": "object",
+      "properties": {
+        "parameterId": {
+          "description": "The unique identifier for the search parameter",
+          "type": "string"
+        },
+        "name": {
+          "description": "The concept name",
+          "type": "string"
+        },
+        "domain": {
+          "description": "The domain for this concept",
+          "type": "string"
+        },
+        "type": {
+          "description": "The vocabulary type",
+          "type": "string"
+        },
+        "group": {
+          "description": "Is the concept a parent or child",
```

Comment:
I don't understand how parent vs child maps to group vs no group. Is this really more of an enum  instead of a boolean?

I'm guessing `group = true` indicates this is a parent property.

Reply:
A parent subsumes children and a child subsumes nothing